gcloud/datastore/test_batch.py

```diff
@@ -454,3 +443,27 @@ def completed_key(self, new_id):
         new_key = self.__class__(self.dataset_id)
         new_key._id = new_id
         return new_key
+
+
+class _Client(object):
```

Comment:
I think I still like mocking it -- there is a lot of extra behavior in the real `Client` class which isn't germane to its use testing `Batch` / `Transaction` / `Query`.